explorer/src/pages/Attestation/AttestationInfo/index.tsx

```diff
@@ -1,23 +1,23 @@
 import { Attestation } from "@verax-attestation-registry/verax-sdk";
+import { Hex, hexToNumber } from "viem";
 
 import { displayAmountWithComma } from "@/utils/amountUtils";
 
-import { createDateListItem } from "./utils";
-import { Hex, hexToNumber } from "viem";
 import { toModuleById } from "@/routes/constants";
 import { cropString } from "@/utils/stringUtils";
-import { DASH } from "@/constants";
+import { Link } from "@/components/Link";
+import { createDateListItem } from "./utils";
 
 export const AttestationInfo: React.FC<Attestation> = ({ ...attestation }) => {
   const { attestedDate, expirationDate, revocationDate, id, revoked, attester, portal, subject } = attestation;
```

Comment:
@alainncls @orbmis 
We've find out that attestedDate, expirationDate, revocationDate came as string, not as number type what declared in OnChainAttestation, could you look into? 
Version sdk 0.0.15 
where you will better to write about some questions like that?

Reply:
Oops! Yes, this is an issue!  
In such cases you can open an issue on the [Project Board](https://github.com/orgs/Consensys/projects/17/views/2) and/or ping us on [Discord](https://discord.gg/Sq4EmYdBEk)!  
Anyway, I'll fix that in a future release of the SDK, promise!